galsim/nfw_halo.py

```diff
@@ -42,7 +42,15 @@ def __init__(self, omega_m=0.3, omega_lam=0.7):
         self.omega_lam = omega_lam
         self.omega_c = (1. - omega_m - omega_lam)
         #self.omega_r = 0
```

Comment:
I assume they were put in as a starter implementation in case we eventually wanted to enable radiation.  I don't think it hurts to leave them here.